src/main/java/mate/project/store/controller/BookController.java

```diff
@@ -28,38 +29,44 @@
 public class BookController {
     private final BookService bookService;
 
+    @PreAuthorize("hasAnyAuthority('ADMIN', 'USER')")
     @GetMapping("/{id}")
     @Operation(summary = "Get book by id", description = "Get book by id")
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
+    @PreAuthorize("hasAnyAuthority('ADMIN', 'USER')")
```

Comment:
As you have
.anyRequest().authenticated()
in config there is no real value for hasAnyAuthority()
Please try with removed annotation, I believe there won't be difference in behaviour